cmd/bootkube/render.go

```diff
@@ -24,16 +24,20 @@ var (
 	}
 
 	renderOpts struct {
-		assetDir    string
-		etcdServers string
-		apiServers  string
-		altNames    string
+		assetDir       string
+		caFile         string
+		privateKeyFile string
+		etcdServers    string
+		apiServers     string
+		altNames       string
 	}
 )
 
 func init() {
 	cmdRoot.AddCommand(cmdRender)
 	cmdRender.Flags().StringVar(&renderOpts.assetDir, "asset-dir", "", "Output path for rendered assets")
+	cmdRender.Flags().StringVar(&renderOpts.caFile, "ca-file", "", "Path to an existing PEM encoded CA.")
+	cmdRender.Flags().StringVar(&renderOpts.privateKeyFile, "private-key-file", "", "Path to the CA RSA private key.")
```

Comment:
s/private-key-file/ca-private-key/

Add behavior notes: "Required if --ca-certificate flag is set."